source/decorated.d

```diff
@@ -135,4 +143,38 @@ unittest
         assert(bar() == "D roks: Hello Decorated");
         assert(baz() == "D roks: Hello Decorated");
```

Comment:
Not related with this PR too, but `roks` -> `rox`

Reply:
done